src/extension.ts

```diff
@@ -2451,7 +2435,7 @@ function getTxtCmd(variable: string, modifiedEnv: { [key: string]: string }) {
   const shellExecutable = path.basename(vscode.env.shell);
   switch (shellExecutable) {
     case "cmd.exe":
-      return `set ${variable}="${modifiedEnv[variable]}"`;
+      return `set "${variable}=${modifiedEnv[variable]}"`;
```

Comment:
so if `modifiedEnv[variable] ` have spaced in them it will be fine?

Reply:
yes the issue was the position of the quote. It was like this on v0.5.1 and before.